src/js/_enqueues/lib/nav-menu.js

```diff
@@ -601,6 +604,26 @@
 					titleEl.text( wp.i18n._x( '(no label)', 'missing menu item navigation label' ) ).addClass( 'no-title' );
 				}
 			} );
+
+			// Show warning when url is empty/invalid in custom menu item type.
+			menuToEdit.on( 'change input', '.edit-menu-item-url', function( e ) {
+
+				const input = $( e.currentTarget );
+				const url = input.val();
+				const urlEl = input.closest( '.menu-item' ).find( '.field-url' );
+
+				if ( '' === url || 'https://' === url || 'http://' === url ) {
+
+					urlEl.addClass( 'form-invalid' );
+					urlEl.attr( 'placeholder', 'https://' );
```

Comment:
We don't need to change the placeholder value in any case. So remove this line from both the If and else conditions.

Reply:
- Resolved in https://github.com/WordPress/wordpress-develop/pull/6389/commits/24598ed85737201ee50112246c4c0b6da3458029 🚀 .